inference/models/starcoder.cc

```diff
@@ -180,6 +180,7 @@ void STARCODER::create_starcoder_model(
   Tensor output;
   if (mode == BEAM_SEARCH_MODE) {
     Tensor softmax = ff.softmax(lm_head, -1);
+    // output = ff.beam_top_k(softmax, startcoder_config.max_beam_width, false);
```

Comment:
same comment as the falcon.cc model above :)

Reply:
same as above.